conf/script/build-ha-io

```diff
@@ -817,8 +817,14 @@ _s3server_rsc_add() {
 
    local count=1
    for i in ${s3server_fids[@]}; do
+      # [KN] According to S3 team, 15 seconds is sufficient for shutting down
+      # Moreover, in s3server systemd the timeout is 7sec so 15sec at Pacemaker should suffice.
+      #
+      # As a side effect, this will force Pacemaker to poll status (while stopping) more often.
+      # According to Pacemaker sources, systemd resources started/stopped and then monitored during 'timeout' 
+      # period until the desired status is reported. Delay between those 'monitor' actions is min(2000, timeout / 10).
       sudo pcs -f $cib_file resource create s3server-$suffix-$count systemd:$i op \
-          stop timeout=600
+          stop timeout=15
```

Comment:
If (for some reason) s3server won't be able to stop in 15 secs - Pacemaker will fence the node. Is this what we want?

Reply:
At systemd level [there is another timeout of 7 seconds](https://github.com/Seagate/cortx-ha/blob/4e59c673f8db0c06c6d5a54e2d6bcb31c7891241/conf/script/build-ha-io#L775) for s3server. Do you think there is a chance that systemd would not manage to kill the process within 15 seconds?

Reply:
Ok then. But still - what will be the polling timeout now after we use 15 seconds? 15/10 == 1.5 seconds instead of 2? What's the percentage that we gain with this for the total failover/back time?

Reply:
Perhaps we need to experiment with this setting first.